### PR TITLE
[Overflow-334] [MARKUP] Implement Delete Modal for Admin Team Members

### PR DESCRIPTION
**Pull request by IanUrriza10**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-344
## Expected Output
* Clicking on the Delete button will prompt the user to confirm the Deletion
* The modal will also have the username as the header

## Screenshots
image

### CODE REVIEW

File: frontend/pages/manage/teams/[slug]/index.tsx [hunk 1]
```diff
@@ -194,23 +205,40 @@ const onPageChange = async (first: number, page: number): Promise<void> => {
...
+        tempMembers.find((item: Member): boolean => item.key === key) // replace temporary data
 
-const renderMembersActions = (key: number): JSX.Element | undefined => {
-    return <div className="flex flex-row justify-end gap-4 ">{deleteAction(key)}</div>
-}
+    const deleteAction = (key: number): JSX.Element => {
+        return (
+            <div
+                onClick={() => {
+                    setIsMemberToDelete(findMember(key)) //
```
Comment (nicoleamber) [inline frontend/pages/manage/teams/[slug]/index.tsx]:
```suggestion
                    setIsMemberToDelete(findMember(key))
```

File: frontend/pages/manage/teams/[slug]/index.tsx [hunk 2]
```diff
@@ -194,23 +205,40 @@ const onPageChange = async (first: number, page: number): Promise<void> => {
         console.log('ok')
     })
 }
-const handleDelete = (event: React.MouseEvent<HTMLElement>): void => {
-    console.log('Delete')
-}
+const MembersTab = (): JSX.Element => {
+    const [isOpenDelete, setIsOpenDelete] = useState<boolean>(false)
+    const [memberToDelete, setIsMemberToDelete] = useState<Member | undefined>()
```
Comment (nicoleamber) [inline frontend/pages/manage/teams/[slug]/index.tsx]:
```suggestion
    const [memberToDelete, setMemberToDelete] = useState<Member | undefined>()
```

**Review by nicoleamber — DISMISSED** [review]

File: frontend/pages/manage/teams/[slug]/index.tsx [hunk 3]
```diff
@@ -194,23 +205,40 @@ const onPageChange = async (first: number, page: number): Promise<void> => {
...
+        return (
+            <div
+                onClick={() => {
+                    setMemberToDelete(findMember(key))
+                    setIsOpenDelete(true)
+                }}
+                className="cursor-pointer"
+            >
+                <Icons name="table_delete" additionalClass="fill-gray-500" />
+            </div>
```
Comment (jules-lucero) [inline frontend/pages/manage/teams/[slug]/index.tsx]:
```suggestion
            <Button
                usage="toggle-modal"
                onClick={() => {
                    setMemberToDelete(findMember(key))
                    setIsOpenDelete(true)
                }}
            >
                <Icons name="table_delete" additionalClass="fill-gray-500" />
            </Button>
```

**Review by jules-lucero — APPROVED** [review]

**Review by nicoleamber — APPROVED** [review]

**Comment by presyos** [comment]
## Approved PR
- Can click the delete icon
   - Delete model is displayed
   - Can click cancel then the modal will disappear
   - Can click x then the modal will disappear
   - Can click outside of the model then the modal will disappear
## Proof

<details>
[Screencast 2023-03-23 13-24-20.webm](https://user-images.githubusercontent.com/112840596/227111859-da209272-2125-43d0-8ae0-dafe819edf6d.webm)



</details>

Test Case:  https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=1860685680
------------------------------------------------------
- Total Test time:  0.25 hr.
- Total Comments: 0
- Total Iterations: 0

**Review by presyos — APPROVED** [review]

**Pull request merged**